utils/fixed_decimal/src/decimal.rs

```diff
@@ -1945,9 +1945,9 @@ impl FromStr for FixedDecimal {
 /// specifies not only the point on the number line but also the precision of the number to a
 /// specific power of 10. This enum augments a floating-point value with the additional
 /// information required by FixedDecimal.
+#[non_exhaustive]
```

Comment:
Note: DoublePrecision is good as `non_exhaustive`.

Reply:
+1